### DISCUSSION TITLE
Welcome to geogram Discussions!

### DISCUSSION THREAD

**Discussion by BrunoLevy** [BrunoLevy/geogram#41] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sebmestrallet** [comment]
Hi there, Sébastien here

I use Geogram for my PhD reseach (polycube-based hexahedral mesh generation). I'm hoping this community will allow for 	mutual aid between users, advices, and showcasing of Geogram's capabilities !

**Comment by BrunoLevy** [comment]
Hi Sébastien, welcome !
Yes, it is the place !
(you can also post an issue if you encounter a problem/bug)
